modules/visual_script/doc_classes/VisualScriptFunctionCall.xml

```diff
@@ -1,55 +1,77 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualScriptFunctionCall" inherits="VisualScriptNode" version="4.0">
 	<brief_description>
+		A Visual Script node for calling a function.
 	</brief_description>
 	<description>
+		[VisualScriptFunctionCall] is created when you add or drag and drop a function onto the Visual Script graph. It allows to tweak parameters of the call, e.g. what object the function is called on.
 	</description>
 	<tutorials>
 	</tutorials>
 	<methods>
 	</methods>
 	<members>
 		<member name="base_script" type="String" setter="set_base_script" getter="get_base_script">
+			The script to be used when [member call_mode] is set to [constant CALL_MODE_INSTANCE].
 		</member>
 		<member name="base_type" type="StringName" setter="set_base_type" getter="get_base_type" default="@&quot;Object&quot;">
+			The base type to be used when [member call_mode] is set to [constant CALL_MODE_INSTANCE].
 		</member>
 		<member name="basic_type" type="int" setter="set_basic_type" getter="get_basic_type" enum="Variant.Type">
+			The type to be used when [member call_mode] is set to [constant CALL_MODE_BASIC_TYPE].
 		</member>
 		<member name="call_mode" type="int" setter="set_call_mode" getter="get_call_mode" enum="VisualScriptFunctionCall.CallMode" default="0">
+			[code]call_mode[/code] determines the target object on which the method will be called. See [enum CallMode] for options.
 		</member>
 		<member name="function" type="StringName" setter="set_function" getter="get_function" default="@&quot;&quot;">
+			The name of the function to be called.
 		</member>
 		<member name="node_path" type="NodePath" setter="set_base_path" getter="get_base_path">
+			The node path to use when [member call_mode] is set to [constant CALL_MODE_NODE_PATH].
 		</member>
 		<member name="rpc_call_mode" type="int" setter="set_rpc_call_mode" getter="get_rpc_call_mode" enum="VisualScriptFunctionCall.RPCCallMode" default="0">
+			The mode for RPC calls. See [method Node.rpc] for more details and [enum RPCCallMode] for available options.
```

Comment:
What is the meaning of RPC?

Reply:
`rpc` description from Node explains it.